client/components/platform-checkout/hooks/use-platform-checkout-user.js

```diff
@@ -0,0 +1,10 @@
+// hook for handling API calls to get and create platform checkout user.
+const usePlatformCheckoutUser = () => {
+	const isRegisteredUser = true;
+
+	return {
+		isRegisteredUser,
```

Comment:
We now have an endpoint for this 🙌 . What do you think about creating a follow-up to make an API call?

Reply:
Creating a follow-up ticket for this sounds good to me. :+1: